### DISCUSSION TITLE
MAA无法与Alas同时运行

### DISCUSSION THREAD

**Discussion by HS7097** [MaaAssistantArknights/MaaAssistantArknights#6401] [Q&A] [answered]
Alas与MAA使用的模拟器都为mumu12，
Alas正在运行时MAA连不上mumu12的任意实例
都会被Alas直接重启ADB
这个该怎么解决呀？

**Comment by ABA2396** [comment]
用的是同一个adb吗？

**Reply by HS7097** [reply]
是的

**Comment by ChingCdesu** [comment]
使用mumu的多开试试，我用mumu12，是可以同时用的，不过要在alas和maa里面分别自己设置模拟器地址

**Reply by HS7097** [reply]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/103177863/df56a62b-4835-4108-800b-a2593ac14d8b)
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/103177863/884c90f2-9b67-4caf-a31f-26a8548d38f0)
mumu12也不行，试过了

**Reply by HS7097** [reply]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/103177863/0b536289-c058-434e-ac0a-ce3c8d9f22c8)
这些都开了

**Reply by ABA2396** [reply, marked as answer]
alas用的不是模拟器的adb，用的是自己的，你要把maa里的adb也选alas用的那个

**Reply by HS7097** [reply]
直接在Alas安装目录内搜索ADB程序然后粘贴到MAA的ADB程序所在路径并替换就好了吗？

**Reply by ABA2396** [reply]
同版本就行，你也可以直接把那个路径填到maa里面。

**Reply by ChingCdesu** [reply]
这个是蓝叠，不是mumu。mumu12和原生的adb都是兼容的

**Reply by HS7097** [reply]
我是從MUMU12換回藍曡的，就是因爲ADB被强行終止

**Reply by ChingCdesu** [reply]
可能版本不对吧，我回去发个我能用的mumu版本号给你

**Comment by aosaki20** [comment]
你用mumu12多开器再开一个模拟器，然后多开器会给你两个不同的adb，你alas和maa用各自的adb就行

**Reply by aosaki20** [reply]
多开器的adb显示在窗口右上角

**Reply by ABA2396** [reply]
不同的是端口，adb要求用一样的，adb启动的时候会关闭不同版本的adb
